baseband/mark4/header.py

```diff
@@ -324,6 +324,8 @@ def _track_assignment(cls, ntrack, bps, fanout):
             return np.concatenate((ta, ta + 32), axis=1)
         elif ntrack == 32:
             return ta
+        elif ntrack == 16:
+            return ta[:, :2, :].reshape(2, 8).T.reshape(4, 2, 2)
```

Comment:
This should be `ta[:, ::2, :]` to be consistent with the sample file, no?

Reply:
I was wrong. This should be `ta[:, ::2, :] // 2` - we need numbers between 0 and 15.